components/Form.js

```diff
@@ -22,39 +27,145 @@ export default function Form({ handleSubmit }) {
         required
       />
       <CharacterCounter maxLength={30} counter={count} />
-      <label htmlFor="description">Description:</label>
-      <textarea
+      <Label htmlFor="description">Description:</Label>
+      <DescriptionTextarea
         id="description"
         name="description"
         maxLength="100"
         onChange={(event) => setCountDescription(event.target.value.length)}
       />
       <CharacterCounter maxLength={100} counter={countDescription} />
-      <label htmlFor="imageFile">Add your sketch:</label>
-      <input
+      {router.pathname === "/create-new-project" && <SelectCategories />}
+      <Label htmlFor="imageFile">Add your sketch:</Label>
+      <Input
         type="file"
         name="imageFile"
         id="imageFile"
         size={10000}
         required
       />
-      <Button type="submit" disabled={statusUpload}>
-        Save
-      </Button>
-      <p>{statusUpload}</p>
+      <ButtonWrapper>
+        <Button type="button" onClick={() => router.back()}>
+          <StyledCancelIcon /> Cancel
+        </Button>
+        <Button type="submit" disabled={statusUpload}>
+          <StyledSaveIcon /> Save
+        </Button>
+      </ButtonWrapper>
+      <LoadingStatus>
+        {statusUpload && <Spinner />}
+        {statusUpload}
+      </LoadingStatus>
     </StyledForm>
   );
 }
 
+const StyledForm = styled.form`
+  display: grid;
+  padding: 0 1em;
+`;
+
+const Label = styled.label`
+  font-weight: 600;
+  font-size: 1.2em;
+  padding: 0.5em 0;
+`;
+
+const styledInputs = css`
+  background-color: var(--color-list-items-white);
+  border: none;
+  border-radius: 2em;
+  padding: 0.7em;
+  &:focus {
+    border-color: var(--color-buttons-yellow);
+  }
+`;
+
+const Input = styled.input`
+  ${styledInputs}
+  min-height: 5vh;
+  &:last-of-type {
+    border: none;
+    background: none;
+    font-size: 0.9em;
+  }
+`;
```

Comment:
Why did you decide to not put these stylings in one styled component? 🤔

Reply:
because the textarea gets the same styling, i just renamed it and would leave it like that :)